osgbuild/promoter.py

```diff
@@ -3,13 +3,16 @@
 
 from __future__ import absolute_import
 from __future__ import print_function
-import os
 import re
 import sys
 try:
     import configparser
 except ImportError:
     import ConfigParser as configparser
+try:
+    from typing import Dict, Set, List
```

Comment:
Are these (`Dict`, `Set`, `List`) actually used?

Reply:
In Python 3 these can be used as annotations to specify types, and in Python 2 you can do the same with magic comments. Both are just advisory but can be used for linting with an IDE or mypy (google for it).

I'm mostly using it for documenting the types so I should probably just use a docstring instead...

Reply:
Yikes, you mean comments that mention `Dict` etc mean something?  I had no idea.

Anyway was just curious since i didn't see any (non-comment) references to them.  But thanks for the explanation..!